controller/test/blower_fsm/blower_fsm_test.cpp

```diff
@@ -80,25 +102,80 @@ TEST(BlowerFsmTest, PressureControl) {
   testSequence({
       // Pressure starts out at PEEP and rises to PIP over period
       // PressureControlFsm::RISE_TIME.
-      {p, /*blower_enabled=*/true, 0, cmH2O(10), ValveState::CLOSED},
-      {p, /*blower_enabled=*/true, rise_time_ms / 4, cmH2O(12.5),
+      {p, r, /*blower_enabled=*/true, 0, cmH2O(10), ValveState::CLOSED},
+      {p, r, /*blower_enabled=*/true, rise_time_ms / 4, cmH2O(12.5),
        ValveState::CLOSED},
-      {p, /*blower_enabled=*/true, rise_time_ms / 2, cmH2O(15),
+      {p, r, /*blower_enabled=*/true, rise_time_ms / 2, cmH2O(15),
        ValveState::CLOSED},
-      {p, /*blower_enabled=*/true, 3 * rise_time_ms / 4, cmH2O(17.5),
+      {p, r, /*blower_enabled=*/true, 3 * rise_time_ms / 4, cmH2O(17.5),
        ValveState::CLOSED},
-      {p, /*blower_enabled=*/true, 1000, cmH2O(20), ValveState::CLOSED},
-      {p, /*blower_enabled=*/true, 1999, cmH2O(20), ValveState::CLOSED},
-      {p, /*blower_enabled=*/true, 2001, cmH2O(10), ValveState::OPEN},
-      {p, /*blower_enabled=*/true, 2999, cmH2O(10), ValveState::OPEN},
-      {p, /*blower_enabled=*/true, 3001, cmH2O(10), ValveState::CLOSED},
-      {p, /*blower_enabled=*/true, 3001 + rise_time_ms / 2, cmH2O(15),
+      {p, r, /*blower_enabled=*/true, 1000, cmH2O(20), ValveState::CLOSED},
+      {p, r, /*blower_enabled=*/true, 1999, cmH2O(20), ValveState::CLOSED},
+      {p, r, /*blower_enabled=*/true, 2001, cmH2O(10), ValveState::OPEN},
+      {p, r, /*blower_enabled=*/true, 2999, cmH2O(10), ValveState::OPEN},
+      {p, r, /*blower_enabled=*/true, 3001, cmH2O(10), ValveState::CLOSED},
+      {p, r, /*blower_enabled=*/true, 3001 + rise_time_ms / 2, cmH2O(15),
        ValveState::CLOSED},
-      {p, /*blower_enabled=*/true, 3001 + rise_time_ms, cmH2O(20),
+      {p, r, /*blower_enabled=*/true, 3001 + rise_time_ms, cmH2O(20),
        ValveState::CLOSED},
   });
 }
 
+TEST(BlowerFsmTest, PressureAssist) {
+  VentParams p = VentParams_init_zero;
+  p.mode = VentMode_PRESSURE_ASSIST;
+  // 20 breaths/min = 3s/breath.  I:E = 2 means 2s for inspire, 1s for expire.
+  p.breaths_per_min = 20;
+  p.inspiratory_expiratory_ratio = 2;
+  p.peep_cm_h2o = 10;
+  p.pip_cm_h2o = 20;
+
+  SensorReadings zero = {.patient_pressure = kPa(0),
```

Comment:
This is used a bunch of times, does it have a default constructor that does the same thing? Or can we extract a constant or utility function?

Reply:
Made it a const for the scope of all blower tests.